war/src/main/webapp/WEB-INF/elasticsearch.yml

```diff
@@ -31,39 +33,28 @@ path.home: es
 #
 path.data: ${exo.data.dir}
 
-# Path to where plugins are installed:
-#
-path.plugins: es/plugins
-
 # Default usage of es embedded addon is for test
 #
-index.number_of_replicas: 0
+#index.number_of_replicas: 0
```

Comment:
The development is in progress. The committed changes are the minimal configuration to make ES embedded works with vers 5.2.2.
The optimization and review comes after, stay tuned ;-)